### PR TITLE
Clean up authz

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
We only have "role" now.

### CODE REVIEW

**Review by yhwen — APPROVED** [review]
LGTM

File: nvflare/fuel/sec/authz.py [hunk 1]
```diff
@@ -22,35 +23,27 @@
...
+        if not isinstance(user, Person):
+            raise ValueError(f"user need to be of type Person but got {type(user)}")
+        if submitter and not isinstance(submitter, Person):
+            raise ValueError(f"submitter need to be of type Person but got {type(submitter)}")
         self.right = right
         self.user = user
         self.submitter = submitter
         self.attrs = {}
-        if not submitter:
+        if submitter is None:
```
Comment (nvidianz) [inline nvflare/fuel/sec/authz.py:46]: The original one is better. It can handle empty string.

Comment (YuanTingHsieh) [reply nvflare/fuel/sec/authz.py:46]: I added the checks in L40-41, so we should be good.

**Pull request merged**